doc/modules/mixture.rst

```diff
@@ -211,49 +232,67 @@ Cons
 
 .. _dpgmm:
 
-DPGMM: Infinite Gaussian mixtures
-=================================
+Dirichlet Process: Infinite Gaussian mixture
+============================================
```

Comment:
Actually I would be in favor of removing this subsection completely and merging the content with the previous section to remove redundancies.